docs/spatial-index-guide.md

```diff
@@ -62,13 +62,16 @@ create table source_index(id BIGINT, latitude long, longitude long) stored by 'c
 'SPATIAL_INDEX.mygeohash.maxLatitude'='20.225281',
 'SPATIAL_INDEX.mygeohash.conversionRatio'='1000000');
 ```
-Note: `mygeohash` in the above example represent the index name.
+Note: 
+   * `mygeohash` in the above example represent the index name.
+   * Columns present in spatial_index table properties cannot be altered
+    i.e., sourcecolumns: `longitude, latitude` and index column: `mygeohash` in the above example.
 
 #### List of spatial index table properties
 
 |Name|Description|
 |-----------------------------------|-----------------------------------------------------------------------------------------|
-| SPATIAL_INDEX | Used to configure Spatial Index name. This name is appended to `SPATIAL_INDEX` in the subsequent sub-property configurations. `xxx` in the below sub-properties refer to index name.|
+| SPATIAL_INDEX | Used to configure Spatial Index name. This name is appended to `SPATIAL_INDEX` in the subsequent sub-property configurations. `xxx` in the below sub-properties refer to index name. Newly created column name is same as that of spatial index name. This column is not allowed in any properties except in SORT_COLUMNS table property.|
```

Comment:
instead of saying newly added column, you can mention Spatial index column which is geoid column, something like this to avoid the confusion

Reply:
done